recipes/qt/6.x.x/conanfile.py

```diff
@@ -1196,6 +1196,9 @@ def _create_plugin(pluginname, libname, plugintype, requires):
                     self.cpp_info.components["QCocoaIntegrationPlugin"].frameworks = [
                         "AppKit", "Carbon", "CoreServices", "CoreVideo", "IOKit", "IOSurface", "Metal", "QuartzCore"
                     ]
+                if self.settings.os in ["Macos", "iOS"]:
+                    # https://github.com/qt/qtbase/blob/v6.5.3/src/gui/CMakeLists.txt#L963
+                    self.cpp_info.components["qtGui"].frameworks.append("Metal")
                 elif self.settings.os in ["iOS", "tvOS"]:
```

Comment:
```suggestion
                if self.settings.os in ["iOS", "tvOS"]:
```
Otherwise this change will prevent creation of `QIOSIntegrationPlugin` on iOS